foundation-models/openai/src/main/java/com/sap/ai/sdk/foundationmodels/openai/OpenAiChatCompletionRequest.java

```diff
@@ -248,6 +252,51 @@ public OpenAiChatCompletionRequest withLogprobs(@Nonnull final Boolean logprobs)
             this.toolChoice);
   }
 
+  /**
+   * Only message generation will be performed without calling any tool.
+   *
+   * @return the current OpenAiChatCompletionRequest instance.
+   */
+  @Nonnull
+  public OpenAiChatCompletionRequest withToolChoiceNone() {
```

Comment:
(Comment)

Not a huge fan of having these duplicate methods signatures, only differing by an apparent argument - that actually is not a real method argument. Apart from the `String` based variant.